src/Tribe/I18n.php

```diff
@@ -190,13 +253,25 @@ public function compile_translations( array $strings, $domains ) {
 
 				// Grab the possible strings for default and any other domain.
 				if ( 'default' === $domain ) {
-					$strings[ $key ][] = __( $value );
-					$strings[ $key ][] = __( strtolower( $value ) );
-					$strings[ $key ][] = __( ucfirst( $value ) );
+					if ($flags & static::COMPILE_INPUT) {
+						$strings[ $key ][] = __( $value );
+					}
+					if ($flags & static::COMPILE_STRTOLOWER) {
+						$strings[ $key ][] = __( strtolower( $value ) );
+					}
+					if ($flags & static::COMPILE_UCFIRST) {
+						$strings[ $key ][] = __( ucfirst( $value ) );
+					}
 				} else {
-					$strings[ $key ][] = __( $value, $domain );
-					$strings[ $key ][] = __( strtolower( $value ), $domain );
-					$strings[ $key ][] = __( ucfirst( $value ), $domain );
+					if ($flags & static::COMPILE_INPUT) {
+						$strings[ $key ][] = __( $value, $domain );
```

Comment:
* :no_entry_sign: The $text arg must be a single string literal, not "$value".
* :no_entry_sign: The $domain arg must be a single string literal, not "$domain".

<sub>hash: 47d465761e59147883a12f6b16fe1374</sub>